Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
+
+### Review panel
+
+The review panel is drawn for every round of applications from the OBF Board members and volunteers based on their availability. 
+Any panelist with a clear *conflict of interest*[1] will recuse themselves from deliberation. 
+
+[1] Conflict of interest will occur when a reviewer comes across an application from an applicant they know (identifiable by the project description or online repository).
+
+### Selection criteria
+
+The selection of the fellowship awardees is based on the criteria described below. 
+Please make sure that you include sufficient information in your application to allow the review panel to assess your eligibility fairly.
+
+***A rubric is currently under development, which will be added to this page upon approval from the OBF board members.***
+
+#### Event and nature of your participation
+
+The review panel will assess how the event that you have selected and the nature of your participation promote open science practices in bioinformatics and/or biological research community.
+
+It's important to keep in mind that the event is scheduled to occur anytime in a one-year window starting one month after the application deadline.
+
+**Event of your choice**
+
+Events that are eligible may include but are not limited to, conferences, workshops, code fests, hackathons, training courses, collaborative sprints, informal meet-ups or other skill-building and networking event. 
+If you intend to attend OBF events (BOSC and BCC) and other events hosted by the member projects, your application may be given a slight preference.
+
+**Nature of your participation**
+
+The review panel will look at the nature of your participation (your role) at the event.
+For example, you may be hosting the event, giving a talk, chairing a session, running a Birds of a Feather discussion, offering a training event, facilitating a sprint (or similar session), leading a project at a code fest/hackathon, presenting a poster, participating in a panel discussion, reviewing abstracts for the event, contribute to the event as a co-organiser, or volunteer to help the organisation committee during the event.
+
+#### Open Source Resources
+
+The review panel may look at your general involvement in the open source development (such as your GitHub contributions to the Open Source project) or open science communities (for example, your previous work advocating for open practices and diversity).
+
+Any information regarding your past participation in the project that you are representing will be useful for evaluating your application as well.
+Therefore, please supply the link of the public repository or online resources that will help the panel in the review process.
+
+This will not be a limiting factor if you are new to bioinformatics and open science and express your enthusiasm and willingness to learn and practice open science principles learned from the event of your choice and OBF community.
+
+#### Promoting diversity and representation
```

Comment:
this is somewhat redundant with the earlier diversity text

Reply:
The above section is about application and this section is about reviewing therefore the repeatition.